packages/next/src/server/lib/clone-response.ts

```diff
@@ -1,3 +1,22 @@
+// @ts-ignore https://nodejs.org/api/stream.html#streamreadableisdisturbedstream
+import { isDisturbed, isErrored } from 'node:stream'
```

Comment:
If I read the docs correctly, these are statics on `Readable`. Can't use `node:` either since it needs to work in Edge runtimes.
```suggestion
import { Readable } from 'stream'
```

Reply:
Thank you, let me update this